src/pages/saksbehandling/søknadsbehandling/vilkår/Vilkår.tsx

```diff
@@ -119,16 +112,15 @@ const Vilkår = (props: { sak: Sak; søker: Person }) => {
                     <Route path={vilkårUrl(Vilkårtype.Formue)}>
                         <Formue
                             behandling={behandling}
-                            søker={props.søker}
                             forrigeUrl={vilkårUrl(Vilkårtype.OppholdIUtlandet)}
+                            søker={props.søker}
```

Comment:
Skal den fortsatt vara kvar? Alla de andra har blitt slettet.

Reply:
Hvorfor trengs den og ikke de andra steg:en? ☺️ hade nog isåfall ikke velat ta in den hvis det var muligt. 😊

Reply:
Navnet til søker brukes som en del av en tekst når det gjelder av skjerming av EPS.

Vi må ha enten endre teksten til å fjerne navnet til søker, eller at denne skal ta den inn fortsatt